### PR TITLE
entity to dataframe fix

### PR DESCRIPTION
**Pull request by DtStry**
corrected a little bug in the entity_type_proto_to_dataframe function.

### CODE REVIEW

**Comment by kmaphoenix** [comment]
@MRyderOC can you please review and provide LGTM

File: src/dfcx_scrapi/core/entity_types.py [hunk 1]
```diff
@@ -77,8 +77,9 @@ def entity_type_proto_to_dataframe(
                 entity_type_dict["entity_value"] = entity.value
                 for synonym in entity.synonyms:
                     entity_type_dict["synonyms"] = synonym
-                    main_df = pd.concat(
-                        [main_df, entity_type_dict], ignore_index=True)
+                    entity_type_df = pd.DataFrame(entity_type_dict,index=[0])
```
Comment (MRyderOC) [inline src/dfcx_scrapi/core/entity_types.py:80]: This is nit-picky, but please use whitespaces as needed to make sure we're following [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html) (3.6 Whitespace).

**Comment by MRyderOC** [comment]
Please make the same changes for the "advanced" mode. Specifically:
https://github.com/GoogleCloudPlatform/dfcx-scrapi/blob/6bb8b70b5660e3d5d416500df8aa9ae5506833b0/src/dfcx_scrapi/core/entity_types.py#L105-L107

and
https://github.com/GoogleCloudPlatform/dfcx-scrapi/blob/6bb8b70b5660e3d5d416500df8aa9ae5506833b0/src/dfcx_scrapi/core/entity_types.py#L110-L112

**Comment by kmaphoenix** [comment]
@DtStry please see above notes from @MRyderOC on changes that need to be made.  

For spacing, quotations and other standards, I suggest installing `pylint` and running the `.pylintrc` file in the base directory to lint your files before submitting.

**Comment by DtStry** [comment]
@kmaphoenix this is fixed, let me know if there's anything else.

**Pull request merged**